src/commands/server/config.ts

```diff
@@ -63,7 +66,32 @@ const config: CommandInt = {
         // Get the next argument as the set type.
         const setType = commandArguments.shift();
 
-        if (configType === "set") {
+        if (configType === "toggle") {
+          if (setType === "thanks") {
+            const shouldThank = await getToggleFromSettings("thanks", guild);
+            await setToggle(guild.id, "thanks", !shouldThank);
+            await message.reply(
+              !shouldThank
+                ? "I have turned the thank feature on."
+                : "I have turned the thank feature off."
+            );
+            return;
+          }
+          if (setType === "levels") {
+            const shouldLevel = await getToggleFromSettings("levels", guild);
+            await setToggle(guild.id, "levels", !shouldLevel);
+            await message.reply(
+              !shouldLevel
+                ? "I have turned the level feature on."
+                : "I have turned the level feature off."
+            );
+            return;
+          }
```

Comment:
You can combine this two cases:

```suggestion
          if (setType === "thanks" || setType === "levels") {
            const toggleSetting = await getToggleFromSettings(setType, guild);
            await setToggle(guild.id, setType, !toggleSetting);
            await message.reply(
              `I have turned the ${setType} feature ${!toggleSetting ? 'on' : 'off'}`
            );
            renturn;
          }
```